packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -182,6 +182,9 @@ export type AzureCommunicationCallWithChatAdapterFromClientArgs = {
     callClient: StatefulCallClient;
     chatClient: StatefulChatClient;
     chatThreadClient: ChatThreadClient;
+    options?: {
```

Comment:
i think we might need to call out this change in storybook as well for `AzureCommunicationCallWithChatAdapterFromClientArgs`

Reply:
Yes thats a good point. I think we created a new section in the storybook to include the factory methods for creating from stateful client, so this can be included in that new section.